Makefile

```diff
@@ -0,0 +1,30 @@
+
+DOCKER ?= $(shell which docker)
+DOC_FILES := \
+	README.md \
+	code-of-conduct.md \
+	ROADMAP.md \
+	implementations.md \
+	bundle.md \
+	runtime.md \
+	runtime-linux.md \
+	config.md \
+	config-linux.md \
+	runtime-config.md \
+	runtime-config-linux.md
```

Comment:
Is maintaining this hard-coded list going to be a hassle?  (ie, would something like `$(wildcard *.md)` also be correct without the same maintainer cost?)